src/main/java/mate/academy/springboot/swagger/services/ProductServiceImpl.java

```
@@ -0,0 +1,46 @@
+package mate.academy.springboot.swagger.services;
+
+import java.math.BigDecimal;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.repo.ProductRepository;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ProductServiceImpl implements ProductService {
+    private final ProductRepository productRepository;
+
+    @Override
+    public Product crete(Product product) {
+        return productRepository.save(product);
+    }
+
+    @Override
+    public Product getById(Long id) {
+        return productRepository.getById(id);
```

let's throw an exception in case product isn't found